tcl/pd_menus.tcl

```diff
@@ -575,6 +575,44 @@ proc ::pd_menus::create_preferences_menu {mymenu} {
         -command {::pd_menus::loadpreferences}
     $mymenu add command -label [_ "Forget All..."] \
         -command {::pd_menus::forgetpreferences}
+		
+	switch -- $::windowingsystem {
+       
+	   "win32" {
+	
+    $mymenu add command -label [_ "Reset .pd files to *this* Pd"] \
+        -command { 
+		
+		if {[file exists "$::sys_libdir/bin/wish86.exe"] == 1} { set wishN "wish86" } \
+		else { set wishN "wish85" }
+		
+		package require registry 
+		
+		set detect2 $::sys_libdir
+		set dirrep [regsub -all "/" ${detect2} "\\"]
+		
+		if {[catch {registry set HKEY_CLASSES_ROOT\\.pd "" "PureData"}  errmsg ]  } {
+		::pdwindow::error "${errmsg} \n"
+		::pdwindow::post "This error is normal. To do this we need to start Pd with \"Administrator Rights\".\nSave your work and close Pd.\nThen right-click on Pd's icon and select \"Run as Administrator\".\nThen try again to \"reset .pd files\".\n"
```

Comment:
i wouldn't call it "normal".
either it is an error, than it is an error. or it is normal, then the error message should be suppressed.
instead i'd use something like:

> "Administrator Permissions" are required to set the file associations.
> Save your work and close Pd.
> Then right-click on Pd's icon and select "Run as Administrator".
> Finally try again to "Open .pd files with this Pd."